src/NRediSearch/Extensions.cs

```diff
@@ -3,20 +3,11 @@
 using System;
 using System.Globalization;
 using StackExchange.Redis;
-using static NRediSearch.Client;
 
 namespace NRediSearch
 {
     public static class Extensions
     {
-        /// <summary>
-        /// Set a custom stopword list
-        /// </summary>
-        /// <param name="options">The <see cref="IndexOptions"/> to set stopwords on.</param>
-        /// <param name="stopwords">The stopwords to set.</param>
-        public static ConfiguredIndexOptions SetStopwords(this IndexOptions options, params string[] stopwords)
```

Comment:
removing an API is a hard break; what is the reasoning for this? there may be better options; for example, to avoid a hard break, if an old API gains a method that was *previously* being provided as an extension, then you should retain the old API as a forward, but just remove the `this` modifier; this means it still works if old IL requests it, but newly compiled code will not use it; it might also be worth adding `[Obsolete]` in that scenario